test/health_checker_unittest.cc

```diff
@@ -22,35 +22,30 @@ class HealthCheckerUnittest : public ::testing::Test {
     return health_checker.IsHealthy();
   }
 
-  void SetIsolationPath(const std::string& isolation_path) {
-    isolation_path_ = isolation_path;
-    std::stringstream stream(
-        "HealthCheckFile: './" + isolation_path_ + "/unhealthy'");
-    config_.ParseConfiguration(stream);
-  }
-
-  std::string isolation_path_;
-  MetadataAgentConfiguration config_;
 };
 
+static const std::string IsolationPath(const std::string& test_name) {
```

Comment:
Why not just have this return `std::istringstream`?

Reply:
+1